### DISCUSSION TITLE
Discount or extend trial for specific customer?

### DISCUSSION THREAD

**Discussion by kurakin-oleksandr** [Kyon147/laravel-shopify#111] [Q&A] [answered]
Hello! I'm wondering if it is possible to grant a discount or at least extend a trial for a specific customer?
Thanks

**Comment by Kyon147** [comment, marked as answer]
The easiest way to do it, is create an entire new plan in the plans table and push them to that plan.

**Reply by kurakin-oleksandr** [reply]
Good idea, thank you!
